File: t9n.api/model/ApiMessage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace t9n.api.model
{
    public class ApiMessage
    {
        public ApiMessage(int httpStatus, string message=null, string moreInfo=null)
        {
            HttpStatus = httpStatus;
            Message = message;
            MoreInfo = moreInfo;

        }
        public int HttpStatus { get; set; }
        public string Message { get; set; }
        public string MoreInfo { get; set; }
        public object Value { get; set; }

    }
}

File: t9n_manager_flutter_client/lib/shared/models/api_message.dart
class ApiMessage {
  int httpStatus = 200;
  String message = "";
  String moreInfo = "";
  dynamic? value;

  ApiMessage(this.httpStatus, this.message, this.moreInfo, this.value);

  factory ApiMessage.fromJson(Map<String, dynamic> json) {
    return ApiMessage(json['httpStatus'] ?? 200, json['message'] ?? '', json['moreInfo'] ?? '', json['value']);
  }

  Map<String, dynamic> toJson() {
    return <String, dynamic>{'httpStatus': httpStatus.toString(), 'message': message, 'moreInfo': moreInfo, 'value':value};
  }
}
